selfdrive/manager/process.py

```diff
@@ -148,14 +151,18 @@ def stop(self, retry=True, block=True):
 
     return ret
 
-  def signal(self, sig):
+  def signal(self, sig: int) -> None:
     if self.proc is None:
       return
 
     # Don't signal if already exited
     if self.proc.exitcode is not None and self.proc.pid is not None:
       return
 
+    # Can't signal if we don't have a pid
```

Comment:
This shouldn't happen in practice since the child classes immediately start the process after creating the object. But doesn't hurt to check in case we change the implementation later.